datadog/dogshell/monitor.py

```diff
@@ -175,6 +184,11 @@ def _post(cls, args):
             "message": args.message,
             "options": options
         }
+        if args.restricted_roles:
+            restricted_roles = sorted(set([rr.strip() for rr in args.restricted_roles.split(",") if rr.strip()]))
+            body["restricted_roles"] = restricted_roles
+        else:
+            restricted_roles = None
```

Comment:
Why is this needed? Doesnt seem to do anything

Reply:
Removed in new commit